systemtests/src/test/java/io/enmasse/systemtest/bases/authz/AuthorizationTestBase.java

```diff
@@ -89,6 +89,7 @@ protected void doTestUserPermissionAfterRemoveAuthz() throws Exception {
         createUser(sharedAddressSpace, user, Group.RECV_ALL_BROKERED.toString(), Group.RECV_ALL_STANDARD.toString());
         assertReceive(user);
         removeUser(sharedAddressSpace, user.getUsername());
+        Thread.sleep(5000);
```

Comment:
Why is this needed?

Reply:
@kornys Hmm, that seems like a keycloak bug to me. I agree we need the workaround for now, but we should ask about the behavior.